app/(main)/league/all/page.tsx

```diff
@@ -56,9 +58,7 @@ const Leagues = (): JSX.Element => {
           ))
         ) : (
           <div className="text-center">
-            <p className="text-lg font-bold">
-              You are not enrolled in any leagues
-            </p>
+            <LoadingSpinner height="max-h-32 h-32" />
```

Comment:
Why is this max and hard set height?

Reply:
I added props to the loader so you can set the height of it to match the element it's replacing. I'll revise this so it takes up 100% of the height and width of what it's replacing by default

Reply:
@shashilo This is outdated, can it be resolved?

Reply:
@shashilo pinging you for a reminder, please advise.